ws-nextjs-app/pages/[service]/new_live/[id]/Post/index.tsx

```diff
@@ -1,59 +1,62 @@
-import React from 'react';
-
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
 import pathOr from 'ramda/src/pathOr';
 import { OptimoBlock } from '#models/types/optimo';
-import headings from '#app/legacy/containers/Headings';
+import Heading from '#app/legacy/containers/Headings';
 import Blocks from '#app/legacy/containers/Blocks';
-import paragraph from '#app/legacy/containers/Paragraph';
-import Text from '#app/components/Text';
+import Paragraph from '#app/legacy/containers/Paragraph';
+import UnorderedList from '#app/legacy/containers/BulletedList';
 import { Post as PostType } from './types';
-
-// temporary solution to render LI/ OL blocks.
-const unorderedList = ({ blocks }: { blocks: OptimoBlock[] }) => {
-  const listItems: (string | null)[] = blocks
-    .map(item =>
-      pathOr<string | null>(
-        null,
-        ['model', 'blocks', 0, 'model', 'text'],
-        item,
-      ),
-    )
-    .filter(text => typeof text === 'string');
-
-  if (listItems.length === 0) return null;
-
-  return (
-    <Text>
-      <ul>
-        {listItems.map((item, index) => (
-          // eslint-disable-next-line react/no-array-index-key
-          <li key={index}>{item}</li>
-        ))}
-      </ul>
-    </Text>
-  );
-};
+import styles from './styles';
 
 const PostHeadings = ({ headerBlocks }: { headerBlocks: OptimoBlock[] }) => {
   const componentsToRender = {
-    headline: headings,
-    subheadline: headings,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    headline: (props: any) => (
+      <Heading
+        {...props}
+        headingLevel={3}
+        fontVariant="sansBold"
+        size="greatPrimer"
+        className="headingStyling"
+        css={styles.postHeading}
+      />
```

Comment:
Just curious why we're using the legacy Heading component rather than just the new `components/Heading` component directly? I see that the legacy component is now making use of the new Heading component under the hood, but would it be better to just use the new directly here?

Reply:
Ah I see! 

Probs one to check if they can do rich text in the headings. They don't seem to be able to do it in Captions for images from what I can see (the Caption component also supports inline links), so we may be able to get away with a much more minimal approach here.

Just more so that we can limit the scope of changes. If its just rendering plain-text, then I feel like using the new Heading component directly could be a good fit here.